### ISSUE TITLE
Bug report: From Hex 0x00 character at EOL

### ISSUE THREAD

**Issue by Isanderthul** [gchq/CyberChef#1353]
**Describe the bug**
Decode the following using the FROM HEX module:
`76656869636C652D6C6F636174696F6E2E636F6D00`
results in the string 
`vehicle-location.com.`
Note that the 0x00 results in an incorrect '.' at the EOL, it would rather be better if 0x00 is handled as ''
<img width="1149" alt="Screenshot 2022-05-16 at 10 09 47" src="https://user-images.githubusercontent.com/3693197/168547981-647f0424-d79c-4059-9b94-06479bb9dc94.png">

* Version: 9.37.3
* Compile time: 14/04/2022 17:10:14 UTC
* User-Agent:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:100.0) Gecko/20100101 Firefox/100.0
* [Link to reproduce](https://gchq.github.io/CyberChef/#recipe=From_Hex('Auto')&input=NzY2NTY4Njk2MzZDNjUyRDZDNkY2MzYxNzQ2OTZGNkUyRTYzNkY2RDAw)

**Comment by mikecat** [comment]
Non-printable characters including 0x00 are displayed as `.`. This looks like not a bug but an intended behavior.
